modules/api/src/main/java/net/countercraft/movecraft/MovecraftRepair.java

```diff
@@ -155,11 +157,13 @@ public boolean saveRegionRepairState(World world, ProtectedRegion region) {
 
     public Clipboard loadCraftRepairStateClipboard(Craft craft, Sign sign) {
         File dataDirectory = new File(plugin.getDataFolder(), "CraftRepairStates");
-        String repairName = craft.getNotificationPlayer().getName();
-        repairName += "_";
-        repairName += ChatColor.stripColor(sign.getLine(1));
+        File playerDirectory = new File(dataDirectory,craft.getNotificationPlayer().getUniqueId().toString());
+        if (!playerDirectory.exists()){
+            return null;
```

Comment:
With above, if the new file doesn't create the directory then it should be created in here.